.xxproject

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<xxProperties/>
```

Comment:
please remove

Reply:
@fkwasowi feel free to add a ```.gitignore``` update and include this file type in another PR. This then wont bother you again.